e2e/ansible/playbooks/hyperconverged/test-scaleup-storage-replicas/test-scaleup-storage-replicas.yml

```diff
@@ -121,17 +121,46 @@
          register: old_replica_name
          delegate_to: "{{groups['kubernetes-kubemasters'].0}}"
 
-
        - name: Get the replica deployment name
          shell: source ~/.profile; kubectl get deployment -n {{ namespace }} | grep rep | awk {'print $1'}
          args:
            executable: /bin/bash
          register: rep_deployment_name
          delegate_to: "{{groups['kubernetes-kubemasters'].0}}"
 
-       #### scale up the stoarge replica using kubectl command ####
+       - name: Get the controller deployment name 
+         shell: source ~/.profile; kubectl get deployment -n {{ namespace }} -l openebs.io/controller=jiva-controller --no-headers | awk {'print $1'}
+         args:
+           executable: /bin/bash
+         register: ctrl_deploy
+         delegate_to: "{{groups['kubernetes-kubemasters'].0}}"
 
-       - name: Scaleup the srorage replicas using kubectl command
+       - name: Obtain the Controller deployment yaml
+         shell: source ~/.profile; kubectl get deployment -n {{ namespace }} {{ ctrl_deploy.stdout }} -o yaml > "{{result_kube_home.stdout }}/{{ ctrl_deployment }}"
+         args: 
+           executable: /bin/bash
+         delegate_to: "{{groups['kubernetes-kubemasters'].0}}"
+
+       - name: Replace the Replication factor value
```

Comment:
Would it help to have a patch file instead of taking a -o yaml o/p and re-applying. This also gives us the flexibility to make other changes if needed.